walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -28,11 +28,20 @@
   // Returns available times for meeting requesters based on the events and its attendees' schedules.
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     Set<String> mandatoryAttendees = new HashSet(request.getAttendees());
-    Set<String> allAttendees = getAllAttendees(events);
+    // All attendees include mandatory and optional attendees.
+    Set<String> allAttendees = new HashSet(mandatoryAttendees);
+    allAttendees.addAll(request.getOptionalAttendees());
+
+    List<Event> eventsList = new ArrayList();
```

Comment:
add a comment; create a new events list that only has events that contain our optional or mandatory attendees